lita-xmpp.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |spec|
   spec.name          = "lita-xmpp"
-  spec.version       = "1.1.0.pre"
+  spec.version       = "1.2.0"
```

Comment:
Wasn't sure what you want done here, but this worked for our own internal needs at the time, so that's what I did :)